examples/pennylane/7_Pulse_control_OQC_Lucy/7_Pulse_control_OQC_Lucy.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>We need to ensure we upgrade the PennyLane plugin to the latest released version before this example is merged (so amazon-braket-pennylane-plugin 1.23.0 or newer).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I dont follow, are notebooks on this github page automatically run somewhere?</p>